src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -528,6 +528,11 @@ public String getPagedListResponseFactoryName(Method method) {
     return privateFieldName(Name.upperCamel(method.getSimpleName(), "PagedListResponseFactory"));
   }
 
+  /** The variable name of the gRPC request object. */
+  public String getRequestObjectName(Method method) {
+    return getNotImplementedString("SurfaceNamer.getRequestObjectName");
+  }
```

Comment:
Some languages use `req`, others use `request`. Wasn't sure which one would make a better sensible default so I left it out.